src/components/migrationToV3.stories.mdx

```diff
@@ -440,4 +431,85 @@ const MyChild = () => {
 ### DefaultTheme Interface
 
 In V3 we rename the `global` property of `DefaultTheme` Interface to `globalStyle` to avoid conflict with reserved keyword `global`.
-When you implement your own theme, you need to use `globalStyle` instead of `global`.
\ No newline at end of file
+When you implement your own theme, you need to use `globalStyle` instead of `global`.
+
+## Device
+
+### useLocalVideo
+
+In V3, we remove `setIsVideoEnabled` from `useLocalVideo` hook.
+
+In V2, We exposed `setIsVideoEnabled` via `useLocalVideo` hook to synchronize the state of `LocalVideo` in the `PreviewVideo` SDK Component as a mitigation of [issue #529](https://github.com/aws/amazon-chime-sdk-component-library-react/issues/529). In JS SDK V3, `startVideoPreviewForVideoInput` and `stopVideoPreviewForVideoInput` do not affect the local video state anymore. So we revert the change in PR #556 (https://github.com/aws/amazon-chime-sdk-component-library-react/pull/556).
+
+### Remove Preset Device Selection Options
+
+In V3, we remove preset device selection options:
+
+- "None", "440 Hz" for audio input device.
+- "None", "Blue" and "SMTP Color Bars" for video input device.
+
+The detailed changes are:
+
+- Remove `appendSampleDevices` from Props of `CameraSelection`, `MicSelection`, `AudioInputControl`, `AudioInputVFcontrol`, and `VideoInputControl`.
+- Remove `additionalDevices` from Props of `useAudioInputs` and `useVideoInputs` hook.
+- Remove `DeviceConfig` type.
+
+Before in 2.x:
+
+```tsx
+// Take `CameraSelection` as an example.
+
+import React from 'react';
+import {
+  MeetingProvider,
+  CameraSelection,
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => {
+  return (
+    <MeetingProvider>
+      {/* If you don't want preset device selection options. */}
+      <CameraSelection appendSampleDevices={true} />
+    </MeetingProvider>
+  );
+};
+```
+
+```jsx
```

Comment:
If using `tsx` in other examples, this should be `tsx` since we show the `DeviceConfig` type or else should be fine to drop it.

Reply:
Updated all markdown language.
So this is only for syntax highlighting, either `jsx` or `tsx` works. I kinda prefer to use `tsx` for all. But it also makes sense to use the accurate one.